src/compatTest/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPluginTests.kt

```diff
@@ -721,13 +725,13 @@ class NexusPublishPluginTests {
         assertReleaseOfStagingRepo()
     }
 
-    //TODO: Move to separate subclass with command line tests for @Option
-    //TODO: Consider switching to parameterized tests for close and release
+    // TODO: Move to separate subclass with command line tests for @Option
+    // TODO: Consider switching to parameterized tests for close and release
     @Test
     fun `should allow to take staging repo id to close from command line without its initialization`() {
         writeDefaultSingleProjectConfiguration()
         writeMockedSonatypeNexusPublishingConfiguration()
-        //and
+        // and
```

Comment:
Can we allow for "no-space after" comments?

Reply:
Now that I understand configuration a little more (trailing commas fix), I might be able to bring that exclusion back, but is it really worth I? The maintenance of the exclusion + the bikeshedding with the configuration of the anti-bikeshedding tool? ;) Is there a tool that relies on this? It looked like a behaviour change, that's why I removed it (there was no explanation).

Reply:
If it is a standard in the Kotlin code, we could use `// and` :).

Reply:
One more reason is a potential conflict with the other PRs :-/. However, the conflict resolution can be handled.

Reply:
Ktlint is one of the default formatters. I guess since you opted for that one, probably should follow the style it prescribes. `// ` in general is common practice in Java and Kotlin at enterprise level.

There are only 3 open PRs, open for ~year or more, so I don't think there's a big problem with that.

Reply:
> There are only 3 open PRs, open for ~year or more

:trollface: